cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -463,13 +465,12 @@ def configure_worker_services(api_servers, dns, cluster_cidr):
     kubelet_opts.add('require-kubeconfig', 'true')
     kubelet_opts.add('kubeconfig', kubeconfig_path)
     kubelet_opts.add('network-plugin', 'cni')
-    kubelet_opts.add('logtostderr', 'true')
     kubelet_opts.add('v', '0')
     kubelet_opts.add('address', '0.0.0.0')
     kubelet_opts.add('port', '10250')
     kubelet_opts.add('cluster-dns', dns['sdn-ip'])
     kubelet_opts.add('cluster-domain', dns['domain'])
-    kubelet_opts.add('anonymous-auth', 'false')
+    kubelet_opts.add('anonymous-auth=false', None)
```

Comment:
I misunderstood how the snap options work. anonymous-auth=false will be ignored because it is not part of the config params. However, if I set it to false it will not make it to the arguments list and it will take the default value of true. Opened this issue: 
 https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/314

I am going to remove the anonymous-auth option for now so as not to cause any confusion.